lib/src/client.dart

```diff
@@ -109,6 +109,8 @@ class Client extends MatrixApi {
 
   final Duration sendTimelineEventTimeout;
 
+  bool _kIsWeb = false;
```

Comment:
why not use default value which is kIsWeb

Reply:
IMO: 
- Priority for mobile
- Update the value in `main.dart`

Reply:
i think this value is not gonna change when the app is running, so it should be final and default kIsWeb also

Reply:
make sure it changes when running.

Reply:
@hoangdat @drminh2807 wdyt?

Reply:
I dont understand of benefit of _kIsWeb or default kIsWeb
is the `_kIsWeb` is `isSupportPlatformWeb` ?